.github/workflows/pr_and_main.yml

```diff
@@ -82,16 +74,17 @@ jobs:
           path: ${{ steps.conan-cache-key.outputs.path }}
           key: conan-${{ matrix.os }}-${{ steps.conan-cache-key.outputs.key }}
 
-      - run: cmake --preset ${{ matrix.cmake-preset }}
+      - run: cmake --preset ci-${{ matrix.os }}
       - run: cmake --build --preset build
       - run: ctest --preset test
```

Comment:
Why doesn't this use tox to build?

Reply:
Because we install dependencies via conda (see PR description). I feel it would be odd to use tox only to run commands?

Reply:
But it does build the docs which require `pandoc`. So what is the difference?

Reply:
Docs also has a lot of requirements from `pip`?

Reply:
Yes, that makes it messy. Why do we even need pandoc?

Also, why did you define a target to build the library in tox?

Reply:
Sphinx needs pandoc.

The library target is defined as convenience for developers.

Reply:
> Sphinx needs pandoc.

To do what? grepping for pandoc does not yield any results.

Reply:
I don't know. It fails without.